src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts

```diff
@@ -44,21 +42,33 @@ export class RgwOverviewDashboardComponent implements OnInit, OnDestroy {
   UserSub: Subscription;
   HealthSub: Subscription;
   BucketSub: Subscription;
+  queriesResults: any = {
+    RGW_REQUEST_PER_SECOND: '',
+    BANDWIDTH: '',
+    AVG_GET_LATENCY: '',
+    AVG_PUT_LATENCY: ''
+  };
+  timerGetPrometheusDataSub: Subscription;
+  timerTime = 30000;
+  readonly lastHourDateObject = {
```

Comment:
Why are we not using this from service itself?

Reply:
The `timerTime` and `LastHourDateObject`?

Reply:
Used that one